app/protocol/flows/blockrelay/handle_relay_invs.go

```diff
@@ -249,6 +250,8 @@ func (flow *handleRelayInvsFlow) processAndRelayBlock(requestQueue *hashesQueueS
 		return err
 	}
 
+	log.Infof("Accepted domainBlock %s via p2p", blockHash)
```

Comment:
Instead, please write: "Accepted block %s via relay"
Also, in the log in submit_block.go, please replace "domainBlock" with just "block"